src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,15 @@
+@mixin hand($name, $time, $angle, $step) {
+  position: absolute;
+  transform-origin: bottom center;
+  animation: $name $time steps($step, end) forwards;
+
+  @keyframes #{$name} {
+    from {
+      transform: rotate(0deg);
+    }
+
+    to {
+      transform: rotate($angle);
```

Comment:
Fix angle. Now it is 30 (need 360)

Reply:
it was 360 when time was 60s. Perhaps, I misunderstood the task. Please explain what does the folowing mean: "For the demo purpose rotate hands during `5s` and `60s`"?